src/utils/getValidLocale.ts

```diff
@@ -12,7 +12,16 @@
 const getValidLocale = (requestedLocale = '') => {
```

Comment:
I noticed this returns undefined for our locales `bi` and `tet`. But I assume we accept that because they're missing from Intl api?

Reply:
yes (now that you say it, not that I tried these locales before) - I don't think there is much we can do (and the few places we use `i18n.t` should work if there is a translation)